ansible/roles/provisioner/tasks/nat.yml

```diff
@@ -0,0 +1,62 @@
+- name: NAT
```

Comment:
A comment explaining what are we doing here with this whole config would be certainly useful